V2.11 (#263)
run-ai/docs: docs/admin/admin-ui-setup/admin-ui-users.md

:octicons-versions-24: Department Admin is available in version 2.10 and later.

1. Login to the Users area of the Run:ai User interface at `company-name.run.ai`.
2. Select the `Users` tab for local users, or the `SSO Users` tab for SSO users.

> 3. On the top right, select "Add New Users".

Reviewer: @jasonnovichRunAI "NEW USER" instead of "Add New Users"

> 3. On the top right, select "Add New Users".

Reviewer: ```suggestion
3. On the top right, select "NEW USER".
```

> 4. Choose a User name and email.

Reviewer: ```suggestion
4. Enter the user's email.
```

> 4. Choose a User name and email.

Reviewer: @jasonnovichRunAI Enter a user email

> 4. Choose a User name and email.

Reviewer: ```suggestion
4. Enter the user email.
```

> 5. Select Roles. More than one role can be selected. Available roles are:

Reviewer: ```suggestion
5. Select Roles. More than one role can be selected. Available roles are: 123
```
